src/libpmemset/pmemset.c

```diff
@@ -21,6 +21,7 @@
 struct pmemset {
 	struct pmemset_config *set_config;
 	struct ravl_interval *part_map_tree;
+	struct pmemset_part_descriptor recent_part;
```

Comment:
I'd rather this previous_part or last_accessed or something.

btw, can't you just use the ravl tree to find whatever element?